tasks/Space Fortress.md

```diff
@@ -0,0 +1,69 @@
+# Summary
+Space Fortress is a video game in which participants control a space ship, and must manage resources, launch attacks, and avoid taking damage. 
+
+# References
+Edwards, Bryan D., et al. "Relationships among team ability composition, team mental models, and team performance." Journal of applied psychology 91.3 (2006): 727.
+Link: https://psycnet.apa.org/buy/2006-07101-018
+
+Donchin, Emanuel. "The learning strategies project: Introductory remarks." Acta Psychologica 71.1-3 (1989): 1-15.
+Link: https://www.sciencedirect.com/science/article/abs/pii/0001691889900024
+
+Mané, Amir, and Emanuel Donchin. "The space fortress game." Acta psychologica 71.1-3 (1989): 17-22.
+Link: http://jtoomim.org/brain-training/the%20space%20fortress%20game.pdf
+
+Arthur Jr, Winfred, et al. "Visual attention: Individual differences in training and predicting complex task performance." Acta Psychologica 88.1 (1995): 3-23.
+Link:https://d1wqtxts1xzle7.cloudfront.net/48887105/Visual_attention_Individual_differences_20160916-3035-1nna77z-with-cover-page-v2.pdf?Expires=1631712486&Signature=aIR1S8JjnNgm2VXZMnUYw0OSy7MEmj55MirK~2WaTcgaxhkSFdPM9ztwEWsobAz9tAa2cNSPnjQ5tsCRW0BVCLfxv5UGEf3AW9AIB0kzgzu0CVTfcgAWUzMpE-V2FCXSk3j6un0yamB~Ck7ZoQ5nN9Zf3GZhfr00~dABNYlnnt1fJMGcrZEUsCK89RM6PTjNrWEtzGfjYigHupOZsIsba0u3ozy2-Y0Psnv5ijPTRzQQsuPaNGIxx6fjofgE-7U16tk3cjuPzTSDeVVpyc4vC4EzVPtr6zlxKoT7CTpJ0HCTNczRcGAyUTl-20q9g2OXq7E1M9QB-CBTUuflnCDI0w__&Key-Pair-Id=APKAJLOHF5GGSLRBV4ZA
+
+
+# Stimuli
+## The visual components
+Sample game screen  
+![image](https://github.com/Watts-Lab/task-mapping/blob/Space-Fortress/images/Space_Fortress_Diagram.png)
+## Materials for alternative versions of the experiment 
+N/A
+
+# Procedure
+## Steps
+Screening of Participants:
+- Participants are subjected to an aiming drill, as described in Donchin and Mané(1989).  The aiming drill is a simplified version of the Space Fortress game.  During the drill, the ship is located in the center of the screen, and has to rotate to shoot at mines that appear one at a time in 24 locations around the screen.  Mines do not move and will disappear if not struck by a missile within 10 seconds.
+- Participants are scored on how quickly and accurately they destroy the mines, and are either kept on for the experiment or dismissed based on their best score of three, two minute aiming drills.
+Training and Testing:
+- Participants are scheduled to participate in 10 one hour training sessions, spread over two consecutive weeks(five days a week, for two weeks).
```

Comment:
Session (day) 1 of each two-week period of the study consisted of a screening session, two baseline Space Fortress games, and the CA-VAT. The order of the Aiming Task (screening), Space Fortress, and the CA-VAT were counterbalanced across subjects. All subjects signed an informed consent and then were randomly
assigned to an order.
>  Half read the Aiming Task and Space Fortress instructions,
including optimal strategies, performed the Aiming Task, played two three-minute
Space Fortress games, with a three minute break between the two games, and
performed CA-VAT. 
> The other half of the subjects were administered the CA-VAT before they performed the Aiming Task and Space Fortress games.

Sessions (days) 2-10 had 8 successive 3-minute practice periods followed by a test sequence of two, 3-minute test games and two, 3-minute rest periods. The order of rest (R) and test (T) was RTRT for half the subjects and TRTR for the other half.
During the rest intervals, subjects completed a detailed survey 1 of their experience with specific video games.

Reply:
I talked with Mark, I was trying to keep the description close to the original experiment, should I add this to the procedure, or should I put it in another section?

Reply:
Maybe you make a sub-section in procedure that says experimental procedure and keep this in?